frappe/contacts/address_and_contact.js

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2018, Frappe Technologies and contributors
```

Comment:
there is already another address_and_contact.js

https://github.com/frappe/frappe/blob/develop/frappe/public/js/frappe/misc/address_and_contact.js